Source/WebCore/Modules/websockets/WebSocketDeflater.cpp

```diff
@@ -200,8 +199,8 @@ bool WebSocketInflater::finish()
 
         m_buffer.grow(bufferSize.value());
         size_t availableCapacity = m_buffer.size() - writePosition;
-        size_t remainingLength = strippedLength - consumedSoFar;
-        setStreamParameter(m_stream.get(), strippedFields + consumedSoFar, remainingLength, m_buffer.data() + writePosition, availableCapacity);
+        size_t remainingLength = strippedFields.size() - consumedSoFar;
```

Comment:
Again, we are performing a method call to get the constant value `4`, to do this calculation. But maybe this is optimized away.